packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/mcp/intelligence_handler.py

```
@@ -305,64 +349,38 @@ async def handle_find_complementary_content(
             logger.info(f"🔍 search_engine type: {type(self.search_engine)}")
             logger.info(f"🔍 search_engine is None: {self.search_engine is None}")
             
-            complementary = await self.search_engine.find_complementary_content(
+            result = await self.search_engine.find_complementary_content(
                 target_query=params["target_query"],
                 context_query=params["context_query"],
                 max_recommendations=params.get("max_recommendations", 5),
                 source_types=params.get("source_types"),
                 project_ids=params.get("project_ids"),
             )
             
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)